canonical-websites/mir-server.io
# Add run script and Jekyll

Helps with #2

Simple Jekyll framework and ./run script to get the site running.

QA
--

`./run`, and go to http://127.0.0.1:8028, and http://127.0.0.1:8028/_status/check
